packages/location-management/src/components/LocationUnitView/index.tsx

```diff
@@ -0,0 +1,226 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Row, Col, Menu, Dropdown, Button, Divider } from 'antd';
+import { SettingOutlined, PlusOutlined } from '@ant-design/icons';
+import LocationUnitDetail, { Props as LocationDetailData } from '../LocationUnitDetail';
+import { Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { OpenSRPService } from '@opensrp/server-service';
+import locationUnitsReducer, {
+  fetchLocationUnits,
+  LocationUnit,
+  reducerName as locationUnitsReducerName,
+} from '../../ducks/location-units';
+import { getAccessToken } from '@onaio/session-reducer';
+import {
+  LOCATION_UNIT_FINDBYPROPERTIES,
+  LOCATION_HIERARCHY,
+  LOCATION_UNIT_GET,
+  URL_LOCATION_UNIT_ADD,
+} from '../../constants';
+import { API_BASE_URL } from '../../configs/env';
```

Comment:
Let's get the base URL from props

Reply:
@haseemVD have created a separate issue to deal with it .. it will be solved separately

Reply:
Created separate [issue](https://github.com/OpenSRP/web/issues/256) for this